conda/models/channel.py

```diff
@@ -391,17 +395,19 @@ def url(self, with_credentials=False):
         return None
 
 
-def prioritize_channels(channels, with_credentials=True, platform=None):
+def prioritize_channels(channels, with_credentials=True, subdirs=None):
```

Comment:
Here we're changing the API for `prioritize_channels()` as well as the `urls()` method on the `Channel` object.  I don't see `prioritize_channels` in exports, so I'm hopeful we'll be ok here with conda-build.  I guess we'll see when the tests run.